configs/fine-tune-llm_v2/aws.yaml

```diff
@@ -0,0 +1,14 @@
+head_node_type:
+  name: head
+  instance_type: m5.2xlarge
+worker_node_types: []
```

Comment:
Recording our offline convo for posterity, this flag will be very confusing in non-OA, since it doesn't show up in the UI anywhere, isn't documented, and may lead to unexpected cost if used incorrectly.

Reply:
ack. this template is for OA only